packet.c

```diff
@@ -131,7 +131,7 @@ static void packet_rcv(uint32_t events, struct epoll_event_handler *h)
 
     dump_packet(buf, cc);
 #endif
-
+    printf("sll_ifindex = %d and cc = %d\n", sl.sll_ifindex, cc);
```

Comment:
so, if you build with the `PACKET_DEBUG` define, will it be sufficient for your use-case?